projects/canopy/src/lib/accordion/accordion-panel-heading/accordion-panel-heading.component.scss

```diff
@@ -41,7 +41,9 @@
     pointer-events: none;
   }
 
-  &--active .lg-accordion__heading-icon {
-    transform: rotateX(180deg);
+  &--active {
```

Comment:
I think it's ok to have this on a single line instead of nested since the `&--active` is not used by anything else 🤔

Reply:
Reverted it to how it was.

Reply:
I think this change belongs in the previous commit: `revert: selection on accordion-panel-heading.scss`